Beta_Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -0,0 +1,1648 @@
+
+    # Parse response into context & content entries
+    if sites:
+        title = 'Sentinel One - Gettin List of Sites \n' + \
+                'Provides summary information and details for all sites that matched your search criteria.'
+        for site in sites:
+            contents.append({
+                'ID': site.get('id'),
+                'Creator': site.get('creator'),
+                'Name': site.get('name'),
+                'Type': site.get('siteType'),
+                'Account Name': site.get('accountName'),
+                'State': site.get('state'),
+                'Health Status': site.get('healthStatus'),
+                'Suite': site.get('suite'),
+                'Created At': site.get('createdAt'),
+                'Expiration': site.get('expiration'),
+                'Unlimited Licenses': site.get('unlimitedLicenses'),
+                'Total Licenses': all_sites.get('totalLicenses'),
+                'Active Licenses': all_sites.get('activeLicenses')
+            })
+            context_entries.append({
+                'ID': site.get('id'),
+                'Creator': site.get('creator'),
+                'Name': site.get('name'),
+                'Type': site.get('siteType'),
+                'AccountName': site.get('accountName'),
+                'State': site.get('state'),
+                'HealthStatus': site.get('healthStatus'),
+                'Suite': site.get('suite'),
+                'CreatedAt': site.get('createdAt'),
+                'Expiration': site.get('expiration'),
+                'UnlimitedLicenses': site.get('unlimitedLicenses'),
+                'TotalLicenses': all_sites.get('totalLicenses'),
+                'ActiveLicenses': all_sites.get('activeLicenses')
+            })
+
+        context['SentinelOne.Sites(val.ID && val.ID === obj.ID)'] = context_entries
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(title, contents, removeNull=True),
+        'EntryContext': context
+    })
+
+
+def get_sites_request(updated_at, query, site_type, features, state, suite, admin_only, account_id, site_name,
+                      created_at, limit, site_ids):
+
+    endpoint_url = 'sites'
+
+    params = {
+        "updatedAt": updated_at,
+        "query": query,
+        "siteType": site_type,
+        "features": features,
+        "state": state,
+        "suite": suite,
+        "adminOnly": admin_only,
+        "accountId": account_id,
+        "name": site_name,
+        "createdAt": created_at,
+        "limit": limit,
+        "siteIds": site_ids
+    }
+
+    response = http_request('GET', endpoint_url, params)
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    if 'data' in response:
+        return response.get('data').get('sites'), response.get('data').get('allSites')
```

Comment:
should probably add check if data is a dict before doing `.get`, just to be on the safe side